tests/acceptance/conftest.py

```diff
@@ -36,6 +36,12 @@ def pytest_addoption(parser):
         default="root",
         help="user to log into remote hosts with (default is root)",
     )
+    parser.addoption(
+        "--ssh-priv-key",
+        action="store",
+        default="",
+        help="Path to SSH private key, when required for login",
```

Comment:
minor: suggested rewording: "Path to _an_ SSH private key .."

Reply:
Ok